charts/kafka/chart/templates/kafka-topic-template.yaml

```
@@ -9,5 +9,8 @@ spec:
   topicName: {{ .name }}
   partitions: {{ .partitions }}
   replicas: {{  .replicas }}
+  {{- if .config }}
+  config: {{ .config | toYaml | nindent 4 }}
+  {{ end }}
```

```suggestion
  {{- end }}
```
I know this is a small thing, but the templated files will look much better if we remove unnecessary white space